src/index.html

```diff
@@ -17,16 +17,40 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="container">
+      <div
+        class="search-bar big"
+        data-qa="big"
+      >
+        <img
+          class="images big"
+          src="/src/images/Search.svg"
+          alt="search-img"
```

Comment:
The `alt` attribute for the image is currently set to 'search-img'. Consider providing a more descriptive text that conveys the purpose of the image, such as 'Search icon'. This will improve accessibility for users relying on screen readers.